packages/roosterjs-content-model/lib/publicApi/utils/paste.ts

```diff
@@ -153,3 +163,17 @@ function createFragmentFromClipboardData(
 
     return fragment;
 }
+
+/**
+ * For block elements that have background color style, remove the background color when user selects the merge current format
+ * paste option
+ */
+const blockElementParser: FormatParser<ContentModelBlockFormat> = (
+    format: ContentModelBlockFormat,
+    element: HTMLElement
+) => {
+    if (element.style.backgroundColor) {
```

Comment:
This can fix the bug when paste text with black background. But what if the text has white color? In that case if we delete the dark block background, the text will become invisible. We may need to either remove text color under it, or do a dark to light conversion

Reply:
Hmm yes, I tested Word Desktop and it seems that they still show the text in white making it look invisible. For example:

Copy source
![image](https://github.com/microsoft/roosterjs/assets/8291124/72d4094a-934f-4e09-ba54-b3c07c36ef37)

Word Content: 
![image](https://github.com/microsoft/roosterjs/assets/8291124/180d9ced-d2fa-4492-a861-f25d711b2cc4)

Reply:
Let me double check with Jorge

Reply:
Checked with Jorge and this should be fine as long as we used the Merge Format option in the PasteOption Plugin. Without merge format we should still should the background color of the Div and the list. 
So I had to do additional changes to add background color to lists and fix an issue when merging format of paragraphs in mergeModel function

Reply:
Sorry, I meant that I added that change in this PR.